### DISCUSSION TITLE
Link content in guide/route-trees for pathless routes section is missed from guide/route-matching

### DISCUSSION THREAD

**Discussion by GYHHAHA** [TanStack/router#1580] [Q&A] [answered]
In [this section](https://tanstack.com/router/latest/docs/framework/react/guide/route-matching#pathless-route-matching), there is a hyperlink to pathless routes in [this page](https://tanstack.com/router/latest/docs/framework/react/guide/route-trees). However it seems that it's missing. So is the section in guide/route-matching deprecated?

**Comment by SeanCassiere** [comment, marked as answer]
It exists, just a broken hyperlink.

It should take you here: https://tanstack.com/router/latest/docs/framework/react/guide/routing-concepts#pathless--layout-routes

Edit: This has since been fixed via #1582.
